src/app/screens/LNURLPay/index.tsx

```diff
@@ -270,14 +273,21 @@ function LNURLPay() {
 
     return (
       <Container maxWidth="sm">
-        <PublisherCard
-          title={navState.origin.name}
-          description={navState.origin.description}
-          lnAddress={getRecipient()}
-          image={navState.origin.icon}
-          isSmall={false}
-        />
+        {navState.origin && (
+          <PublisherCard
+            title={navState.origin.name}
+            description={navState.origin.description}
+            image={navState.origin.icon}
+            isSmall={false}
+          />
+        )}
         <dl className="shadow bg-white dark:bg-surface-02dp mt-4 pt-4 px-4 rounded-lg mb-6 overflow-hidden">
+          {getLnAddress() && (
+            <>
+              <Dt>LN Address</Dt>
```

Comment:
If it's no problem space-wise I'd suggest to not abbreviate things and just write `Lightning Address`.